### ISSUE TITLE
Accordion touch targets are too close together for touch users

### ISSUE THREAD

**Issue by christopherthomasdesign** [alphagov/govuk-frontend#2608]
## Related component
Accordion – raised by a user on Slack.

## Context
On accordions, the open section background sits very very close to the first link in the list. This might be a usability issue because there isn't much space between the two touch targets.

<img width="320" alt="Screenshot 2022-04-29 at 12 15 12" src="https://user-images.githubusercontent.com/19834460/165955075-e25f970e-1bce-4b1c-bcd6-eaa0416d3137.png">

## Alternatives
To fix it we could:
* add a margin to the bottom of the section
* add some padding to the top of the first link
* something else?

## Additional information (if applicable)
N/A
